### PR TITLE
Apply prettier on blueprint files

### PR DESCRIPTION
**Pull request by simonihmig**
So they don't cause linting errors

### CODE REVIEW

File: blueprints/@kaliber5/k5-ember-boilerplate/index.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-'use strict';
+"use strict";
```
Comment (lolmaus) [inline blueprints/@kaliber5/k5-ember-boilerplate/index.js:1]: Why double quotes?

Comment (simonihmig) [reply blueprints/@kaliber5/k5-ember-boilerplate/index.js:1]: Was not intentional, probably didn't have the right prettier config applied. Will fix.

Comment (simonihmig) [reply blueprints/@kaliber5/k5-ember-boilerplate/index.js:1]: Oh, I should have looked at PR's diff myself, that's all over the place... 🤦‍♂️

Comment (simonihmig) [reply blueprints/@kaliber5/k5-ember-boilerplate/index.js]: Fixed

**Pull request merged**